contrib/cirrus/CIModes.md

```diff
@@ -42,72 +42,33 @@ of this document, it's not possible to override the behavior of `$CIRRUS_PR`.
 
 ## Cirrus Task contexts and runtime modes
 
-### Intended general PR Tasks (*italic*: matrix)
-+ *build*
-+ validate
-+ bindings
-+ swagger
-+ *alt_build*
-+ osx_alt_build
-+ freebsd_alt_build
-+ docker-py_test
-+ *unit_test*
-+ apiv2_test
-+ *compose_test*
-+ *local_integration_test*
-+ *remote_integration_test*
-+ *container_integration_test*
-+ *rootless_integration_test*
-+ *local_system_test*
-+ *remote_system_test*
-+ *rootless_remote_system_test*
-+ *buildah_bud_test*
-+ *rootless_system_test*
-+ rootless_gitlab_test
-+ *upgrade_test*
-+ meta
-+ success
-+ artifacts
-
-### Intended for PR w/ "release" or "bump" in title:
-+ (All the general PR tasks above)
-+ release_test
-
-### Intended `[CI:DOCS]` PR Tasks:
-+ *build*
-+ validate
-+ swagger
-+ meta
-+ success
-
-### Intended `[CI:BUILD]` PR Tasks:
-+ *build*
-+ validate
-+ *alt_build*
-+ osx_alt_build
-+ freebsd_alt_build
-+ meta
-+ success
-+ artifacts
-
-### Intended `[CI:MACHINE]` PR Tasks:
-
-If and only if the PR is in **draft-mode**, run only the following
-tasks.  The draft-mode check is necessary to remove the risk of
-merging a change that affects the untested aspects of podman.
-
-+ *build*
-+ validate
-+ *alt_build*
-+ win_installer
-+ osx_alt_build
-+ podman_machine_task
-+ podman_machine_aarch64_task
-+ podman_machine_windows_task
-+ podman_machine_mac_task
-+ meta
-+ success
-+ artifacts
+By default cirrus will trigger task depending on the source changes.
+As such a user does not have to use any special titles under normal circumstances.
+
+It is implemented using the `only_if` and field for the cirrus tasks, this logic
+uses the following main rules:
+ - Never skip on cron runs: `$CIRRUS_PR == ''`
+ - Never skip when using the special `CI:ALL` title: `$CIRRUS_CHANGE_TITLE =~ '.*CI:ALL.*'`, see below.
+ - Never skip when a danger files is changed, these file contain things that can
+   effect any tasks so the code cannot skip it. It includes
+   - `.cirrus.yml` (cirrus changes)
+   - `Makefile` (make targets are used to trigger tests)
+   - `contrib/cirrus/**` (cirrus scripts to run the tests)
+   - `vendor/**` (dependency updates)
+   - `hack/**` (contains scripts used by several tests)
+   - `version/rawversion/*` (podman version changes, intended to ensure all release PRs test everything to not release known broken code)
+
+After that test specific rules are added, see the commit in .cirrus.yml for them.
```

Comment:
Perhaps slightly more readable with a comma and a hyphen modifying the compound adjective: "After that, test-specific rules ..."